preparation.sh

```diff
@@ -1,16 +1,25 @@
 #!/usr/bin/env bash
+set -e
 
 export REBUILD_GRAPHS=true
-export ors.engine.preparation_mode=true
-export ors.services.routing.mode=preparation
+export ORS_ENGINE_PREPARATION_MODE=true
+export ORS_SERVICES_ROUTING_MODE=preparation
+
+mv files/data.osm.pbf original.pbf
+osmconvert -v original.pbf --parameter-file=/parameter-file -o=files/data.osm.pbf
+rm -fr graphs/*
```

Comment:
question(non-blocking): just curious, `graphs/` what is this/where did it come from?

Reply:
Is this just when rerunning the command locally not in `Dockerfile`?

Reply:
good catch! I thought I was seeing an issue where there was existing data in there, but I'm not seeing it now so I'll remove it.